src/tests/stdlib/make_executable.ab

```diff
@@ -4,10 +4,10 @@ import { make_executable } from "std/fs"
 // created
 
 main {
-    let tmpdir = unsafe $mktemp -d /tmp/amber-XXXX$
-    unsafe $touch {tmpdir}/amber-symbolic$
+    let tmpdir = trust $mktemp -d /tmp/amber-XXXX$
+    trust $touch {tmpdir}/amber-symbolic$
     if make_executable("{tmpdir}/amber-symbolic") {
         echo "created"
     }
-    unsafe $rm -fr {tmpdir}$
+    trust $rm -rf {tmpdir}$
```

Comment:
I see you changed `-fr` to `-rf`. I was tempted to do this myself, but didn't want to step on anyone's toes. But if you're doing it here, consider changing throughout?

Reply:
the difference between `rf` and `fr` is just a code style.

I think that right now is not so much important

Reply:
Woah! I really don't know when I have written this. This must be some leftover from other branch that I was testing or sth. Sorry for this mess. I'll clean it up later today.